cmd/manager/main.go

```diff
@@ -194,14 +195,18 @@ func main() {
 		setupLog.Error(err, "unable to create CA certificate pool")
 		os.Exit(1)
 	}
-	unpacker := &source.ImageRegistry{
-		BaseCachePath:   filepath.Join(cachePath, "unpack"),
-		CertPoolWatcher: certPoolWatcher,
```

Comment:
Loss of `CertPoolWatcher: certPoolWatcher` here...

Reply:
Not seeing `ContainersImageRegistry` use certificate, so is this necessary?

Reply:
Yes, in BuildHTTPClient

Reply:
Yeah, the containers/image source implementation lets us set the CA files on a context struct, which I think are reloaded on every pull.

And yeah, the certPoolWatcher _should_ still be used to connect to catalogd.